packages/emitter/src/index.ts

```diff
@@ -25,12 +45,33 @@ function escapeQuotes(value: string, quote: Quote) {
   return value.replace(NEEDS_ESCAPING[quote], "\\$&");
 }
 
-function escapeValue(value: string, quote: Quote) {
-  if (value === "") {
-    return quote + quote;
+const QUOTE_COUNTER: { [Q in Quote]: RegExp } = {
+  '"': /"/g,
+  "'": /'/g,
+};
+
+function countQuote(value: string, quote: Quote) {
+  return value.match(QUOTE_COUNTER[quote])?.length ?? 0;
```

Comment:
Could we implement this without regexp? Like:
```typescript
let count = 0;
for (let i = 0; i < value.length; ++i) {
  if (input[i] === quote) {
     count++; 
  }
}
return count;
```
I know that performance difference will be marginal, but this library has been built with high performance in mind, so I would prefer to use simple for loop when possible :)